interface.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+
+def choose_working_mode():
+    """
+    Function has input with common checks
+    :return: (string) function returns each string input
+    """
+    i = 0
```

Comment:
I'd call this `attempt` instead of `i`. It would make the code much more readable.

Reply:
fixed